ethereum/scripts/configure-beefy.ts

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();
+const hre = require("hardhat");
+
+const SubClient = require('../../test/src/subclient').SubClient;
```

Comment:
I'd rather not pull in E2E test fixture stuff. Its also not necessary.

Instead can you create a Polkadot-API instance using the example at: https://polkadot.js.org/docs/api/start/create.

For determining the API endpoint you can pass an environment variable to the script. This will allow the script to work both for local and staging environments.

Reply:
fixed https://github.com/Snowfork/snowbridge/pull/487/commits/ceeefc1b38af8e0510d0244fb44dedc747482523